llvm/test/MC/X86/tls.s

```diff
@@ -0,0 +1,37 @@
+// RUN: llvm-mc -filetype=obj -triple=x86_64 %s -o %t
+// RUN: llvm-objdump -dr --no-print-imm-hex %t | FileCheck %s
+
+// CHECK:      <_start>:
```

Comment:
Why not use the existing test file llvm/test/MC/ELF/relocation.s?

Reply:
There were no checks for both instructions and relocation types emitted by llvm-mc, so I added new test file for this R_X86_64_CODE_4_GOTTPOFF type and other relocation types to be added.

Reply:
Why do we need to check the instructions? There should be existing encoding/decoding tests there.

IMHO, checking the similar relocation types in a single file can help use not miss some checks?

Reply:
It's not to check encoding/decoding. It showed the map of instruction and relocation information (type, symbol and offset), and that's the output what users got from llvm-mc/llvm. Some tests in llvm/test/MC folder did same thing.

I'm okay with adding the check of relocation information in relocation.s and not adding this test file, if you insist.

Reply:
I've removed the test file newly added and add the check in relocation.s.